Content.Server/Medical/Components/HealingComponent.cs

```diff
@@ -31,6 +41,14 @@ async Task<bool> IAfterInteract.AfterInteract(AfterInteractEventArgs eventArgs)
             {
                 return true;
             }
+            else
+            {
+                var target = eventArgs.Target.GetComponent<DamageableComponent>();
```

Comment:
Just use TryGetComponent, it does both in one and is inherently faster

Reply:
See change in 69cb73c.